src/collects/seashell/backend/runner.rkt

```diff
@@ -323,8 +340,11 @@
                   ;; Behaviour is inconsistent if we just exec directly.
                   ;; This seems to work.  (why: who knows?)
                   (putenv "ASAN_OPTIONS"
-                          "allocator_may_return_null=1:detect_leaks=1:stack_trace_format=\"{'frame': %n, 'module': '%m', 'offset': '%o', 'function': '%f', 'function_offset': '%q', 'file': '%s', 'line': %l, 'column': %c}\"
-                            detect_stack_use_after_return=1")
+                          "allocator_may_return_null=1:
+                           detect_leaks=1:
+                           log_path='/tmp/seashell-asan':
+                           stack_trace_format='{\"frame\": %n, \"module\": \"%m\", \"offset\": \"%o\", \"function\": \"%f\", \"function_offset\": \"%q\", \"file\": \"%s\", \"line\": %l, \"column\": %c}'
```

Comment:
Maybe rework your regexes (as you're doing this already) to work with the standard ASAN stack output format, instead of this hacked-together JSON format.  It's not a big deal, but it'll probably make testing easier with just stock ASAN output.